app/helpers/user_helper.php

```diff
@@ -1,7 +1,7 @@
 <?php
-session_start();
 
-function objectToArray($obj){
+function objectToArray($obj)
+{
     if (!is_object($obj)) {
         return false;
```

Comment:
added this line of code to return array if it is already and array and return false if it is not an object or array
```php
if (is_array($obj)) {
    return $obj;
}